docs/mllib-feature-extraction.md

```diff
@@ -527,6 +527,33 @@ val transformedData2 = parsedData.map(x => transformer.transform(x))
 
 {% endhighlight %}
 </div>
+
+<div data-lang="java">
+{% highlight java %}
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.apache.spark.mllib.feature.ElementwiseProduct;
+import org.apache.spark.mllib.linalg.Vector;
+import org.apache.spark.mllib.linalg.Vectors;
+
+// Create some vector data; also works for sparse vectors
+JavaRDD<Vector> data = sc.parallelize(Arrays.asList(
```

Comment:
As @srowen  pointed out, it would be nice for this to load the same data as the Scala/Python examples (and be as similar as possible, in general).